### ISSUE TITLE
Package warnings on Python 3.8 with minimum pkg levels

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#2153]
On Python 3.8 with PACKAGE_LEVEL=minimum:
```
ERROR: tox 2.0.0 has requirement pluggy<0.4.0,>=0.3.0, but you'll have pluggy 0.13.0 which is incompatible.
ERROR: ipython 7.10.0 has requirement prompt-toolkit<3.1.0,>=2.0.0, but you'll have prompt-toolkit 1.0.15 which is incompatible.
```

On Python 2.7 with latest:
```
ERROR: flake8 3.7.9 has requirement pyflakes<2.2.0,>=2.1.0, but you'll have pyflakes 2.2.0 which is incompatible.
```

**Comment by andy-maier** [comment]
* tox/pluggy:
  * The highest version of tox that still supports py2.6 is 2.9.1, which requires pluggy<1.0,>=0.3.0.
  * pluggy is also pulled in by pytest, whose minimum version 4.3.1 requires pluggy>=0.7.

**Comment by andy-maier** [comment]
Addressed by PR #2158 for master and TBD for 0.17.1.

**Comment by andy-maier** [comment]
More conflicts:
```
ERROR: tox 3.14.0 has requirement importlib-metadata<1,>=0.12; python_version < "3.8", but you'll have importlib-metadata 1.1.3 which is incompatible.
```
